client/src/components/advancedSearch/DateRangeFilter.tsx

```diff
@@ -183,12 +203,16 @@ export const deserialize = ({ queryKey }, query, key) => {
   const filterValue = {}
 
   if (query[startKey]) {
```

Comment:
Suggest to change to `if (Object.hasOwn(query, startKey)) {` so `0` is also valid.